docs/connect.html

```diff
@@ -313,7 +313,13 @@ <h4><a id="connect_rest" href="#connect_rest">REST API</a></h4>
         <li><code>DELETE /connectors/{name}</code> - delete a connector, halting all tasks and deleting its configuration</li>
         <li><code>GET /connectors/{name}/topics</code> - get the set of topics that a specific connector is using since the connector was created or since a request to reset its set of active topics was issued</li>
         <li><code>PUT /connectors/{name}/topics/reset</code> - send a request to empty the set of active topics of a connector</li>
-        <li><code>GET /connectors/{name}/offsets</code> - get the current offsets for a connector (see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-875%3A+First-class+offsets+support+in+Kafka+Connect">KIP-875</a> for more details)</li>
+        <li>Offsets management REST APIs (see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-875%3A+First-class+offsets+support+in+Kafka+Connect">KIP-875</a> for more details):
+            <ul>
+                <li><code>GET /connectors/{name}/offsets</code> - get the current offsets for a connector</li>
+                <li><code>DELETE /connectors/{name}/offsets</code> - reset the offsets for a connector. The connector must exist and must be in the stopped state.</li>
+                <li><code>PATCH /connectors/{name}/offsets</code> - alter the offsets for a connector. The connector must exist and must be in the stopped state. The request body should be a JSON object containing a JSON array <code>offsets</code> field, similar to the response body of the <code>GET /connectors/{name}/offsets</code> REST API.</li>
```

Comment:
Same nit RE trailing period, and same question about linking to the docs for the `PUT /connectors/{name}/stop` endpoint:
```suggestion
                <li><code>PATCH /connectors/{name}/offsets</code> - alter the offsets for a connector. The connector must exist and must be in the stopped state. The request body should be a JSON object containing a JSON array <code>offsets</code> field, similar to the response body of the <code>GET /connectors/{name}/offsets</code> endpoint</li>
```

Reply:
Also, do you think we can include either a link to the generated OpenAPI docs for this endpoint, or a fleshed-out example of a request for this endpoint?

I like that we're calling out the symmetry between the request format for this endpoint and the response format for the `GET /connectors/{name}/offsets` endpoint, but I suspect it may be easier on users if we also provide examples for altering the offsets of a sink and source connector (including some null partitions to denote resets).

Possibly creeping out of scope for this PR, but we might have an additional docs section that goes into detail on managing offsets with connectors (including example requests/responses) and then link to that section from each of these list items, instead of trying to put all of the information for these endpoints into this section. This is definitely not a blocker, though.

Reply:
The link to the generated OpenAPI docs are at the end of this section so I don't think it's necessary to add the same link here as well. Also, I think an actual example might be more helpful than the generated OpenAPI spec where the finest level of granularity is the `ConnectorOffset` schema describing the `partition` and `offset` keys having JSON object values. I was hoping that the link to the KIP should be sufficient, but I do see the value of including actual examples directly in the docs as well.

Reply:
The example is great, thanks 👍